src/components/humanComponent/humanComponent.js

```diff
@@ -0,0 +1,51 @@
+import React, { Component } from 'react';
+import BoardComponent from '../boardComponent/boardComponent';
+import GameHeader from '../gameHeaderComponent/gameHeaderComponent';
+import '../boardComponent/BoardComponent.css';
+import Board from '../../board/board';
+import PhoenixApi from '../../services/phoenixAPI';
+import Statuses from '../gameHeaderComponent/gameHeaderEnum';
+
+class HumanComponent extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            board: new Board(),
+            value: 'X',
+            gameStatus: Statuses.IN_PROGRESS,
+          };
+    }
+
+toggleMarker = async (event) => {
+    const squareIndex = event.target.id;
+    let board = this.state.board;
+    board.makeMark(squareIndex,this.state.value);
+    this.setState({
+      value: this.state.value === 'X' ? 'O' : 'X',
+    });
+    let next_player = this.state.value === 'X' ? 'O' : 'X';
+    let current_player = this.state.value;
```

Comment:
Whenever you set state based on the current state, it is generally recommended to use `setState` passing in a callback. This avoids potential race conditions.

The callback is a function that takes the previous/current state and return the updated state. For example, lets say you are incrementing a counter.

For example, instead of this:
```javascript
this.setState({
  counter: this.state.counter + 1
});
```

Do this
```javascript
this.setState((previousState) => {
  return {
    counter: previousState.counter + 1
  }
});
```

Reply:
Also, here is a good linting rule to enable related to this:

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-access-state-in-setstate.md